src/main/java/online/talkandtravel/model/dto/country/CountryInfoWithUnreadMessagesDto.java

```diff
@@ -0,0 +1,8 @@
+package online.talkandtravel.model.dto.country;
+
+public record CountryInfoWithUnreadMessagesDto(
+    String name,
+    String flagCode,
+    Long unreadMessagesCount
```

Comment:
this is defenetly incorrect. I ask you to forget about Country=Chat.
Country is just country and nothing more. But Chat has attributes as 'messages' and their counts. It is impossible to exist this DTO in current architecture